agent/task.go

```diff
@@ -1,12 +1,12 @@
 package agent
 
 import (
-	"reflect"
 	"time"
 
 	"github.com/docker/swarmkit/agent/exec"
 	"github.com/docker/swarmkit/api"
 	"github.com/docker/swarmkit/log"
+	"github.com/docker/swarmkit/manager/dispatcher"
```

Comment:
I suggest `github.com/docker/swarmkit/apiutils`.

Normally this would be a subpackage under `api`, but I believe @aluzzardi has asked that only generated code be placed under `api`.

ping @aluzzardi @stevvooe